### PR TITLE
Feat/multiple vaults

### PR DESCRIPTION
**Pull request by GopherJ**
close: #962

### CODE REVIEW

**Review by ETeissonniere — APPROVED** [review]
lgtm. Just wondering why we need to identify the vault per cToken instead of ParaId. We could simply overwrite the previous vault of it is failed. Nonetheless, it works as it is so approved it anyways.

**Comment by GopherJ** [comment]
@ETeissonniere because people can be slow to claim back their tokens actually

File: pallets/crowdloans/src/weights.rs [hunk 1]
```diff
@@ -65,62 +65,62 @@ pub trait WeightInfo {
 pub struct SubstrateWeight<T>(PhantomData<T>);
 impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
     fn create_vault() -> Weight {
-        (39_000_000 as Weight)
```
Comment (i5possible) [inline pallets/crowdloans/src/weights.rs:68]: Maybe a stupid question: How is the weight is calculated or set?

**Pull request merged**